_posts/en/newsletters/2021-07-21-newsletter.md

```diff
@@ -0,0 +1,157 @@
+---
+title: 'Bitcoin Optech Newsletter #158'
+permalink: /en/newsletters/2021/07/21/
+name: 2021-07-21-newsletter
+slug: 2021-07-21-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes recent changes to services and client
+software, discusses why wallets should wait before generating taproot
+addresses, lists new software releases and release candidates, and
+summarizes notable changes to popular Bitcoin infrastructure software.
+
+## News
+
+*No significant news this week.*
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+- **Lightning-powered news site Stacker News launches:**
+  [Open source][stacker news github] news site [Stacker News][stacker.news]
+  launched allowing LNURL authentication as well as voting and commenting using
+  LN micropayments.
+
+- **Suredbits announces DLC wallet alpha release:**
+  [Suredbits' bitcoin-s][suredbits blog] software includes a GUI and allows the
+  execution of Discreet Log Contracts (DLCs) on the Bitcoin blockchain using an
+  oracle. The announcement concludes by mentioning they also plan to use [schnorr
+  signatures][topic schnorr signatures] and [Point Time Locked Contracts
+  (PTLCs)][topic ptlc] to implement [DLCs on Lightning][suredbits blog dlcs ln].
```

Comment:
I'm going to change these last few words to "DLCs compatible with LN".  What they're describing can lead to DLCs being routed over LN, but as the last lines of the blog post you link to indicate, this conflicts with the way capital is deployed on LN.  I'm personally unaware of any solution to this problem, so I'd like to slightly tone down the enthusiasm here.

As always, please feel free to revert if you disagree.

Reply:
Yes, good nuance, thanks for the change.